state-synchronizer/src/executor_proxy.rs

```diff
@@ -156,7 +157,13 @@ impl ExecutorProxyTrait for ExecutorProxy {
             intermediate_end_of_epoch_li,
         )?;
         timer.stop_and_record();
-        self.publish_on_chain_config_updates(reconfig_events)
+        if let Err(e) = self.publish_on_chain_config_updates(reconfig_events) {
+            error!(
+                "[state sync] failed to publish reconfig notification in execute_chunk: {}",
+                e
+            );
+        }
```

Comment:
I don't think this function returns a result any more, so you can probably just update the function signature

Reply:
there is an instance earlier in this function where we return result (storage-fetch configs): https://github.com/libra/libra/blob/master/state-synchronizer/src/executor_proxy.rs#L199
I thought about grouping all the per-subscription publishing errors and returning that as a single error so we can unify the location of logging the failure of this fn call here... but it seemed a bit ugly/hassle so making the tradeoff for simplicity, but lmk if you think otherwise!